python-libraries/nanover-core/tests/trajectory/test_frame_data.py

```diff
@@ -737,6 +737,17 @@ def test_copy_doesnt_mutate_original_array(value):
     assert frame.arrays["sample.new"][0] != "baby yoda"
 
 
+@pytest.mark.parametrize("typed_array", [[0], [0.0], ["string"]])
```

Comment:
I guess this is to make sure the array is created with a type? If so it would be good to have a comment about it.

Reply:
Hopefully my added comments clear this up?